### ISSUE TITLE
rule-3 of comment alignment should not work for #endif's

### ISSUE THREAD

**Issue by mihaipopescu** [uncrustify/uncrustify#1287]
Expected versus output diff:
---
```diff
diff --git "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60030-UNI-21727.cpp" "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60030-UNI-21727.cpp"
index c2b6711..645ea21 100644
--- "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60030-UNI-21727.cpp"
+++ "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60030-UNI-21727.cpp"
@@ -1,10 +1,10 @@
 void foo()
 {
 #if defined(SUPPORT_FEATURE)
     bar();
 #endif // SUPPORT_FEATURE
-    // Handle error
+       // Handle error
     if (error != 0)
     {
     }
 }
```

Debug support files:
---
[60030-UNI-21727.cpp](https://gist.githubusercontent.com/mihaipopescu/2bab90a2932dfbf879ce21a44aa38fa8/raw/c2b6711439c0ce65e480bb0a11d3ce734074efb6/60030-UNI-21727.cpp)
[60030-UNI-21727.cpp.log](https://gist.githubusercontent.com/mihaipopescu/2bab90a2932dfbf879ce21a44aa38fa8/raw/726cb5b92a0d37ce52d54d34e8070a420ab8de06/60030-UNI-21727.cpp.log)
[60030-UNI-21727.cpp.unc](https://gist.githubusercontent.com/mihaipopescu/2bab90a2932dfbf879ce21a44aa38fa8/raw/1d7488da376a0c4eed260b6b7f57233643dfa5d0/60030-UNI-21727.cpp.unc)
[Uncrustify.Cpp.cfg](https://gist.githubusercontent.com/mihaipopescu/2bab90a2932dfbf879ce21a44aa38fa8/raw/23d62d9dba5caf352fdc1793792e98eed4b81929/Uncrustify.Cpp.cfg)
[UNI-21727.cpp](https://gist.githubusercontent.com/mihaipopescu/2bab90a2932dfbf879ce21a44aa38fa8/raw/c2b6711439c0ce65e480bb0a11d3ce734074efb6/UNI-21727.cpp)

Disclaimer:
---
Note: This is a generated issue from our internal issue tracker.
Expect a dry description which could be insufficient to properly understand the issue.
On the other hand, the support files should be sufficient to easily reproduce and debug the issue,
but please comment if it's still unclear and I will get back with clarifications.
See main discussion thread here #1223.

**Comment by mihaipopescu** [comment]
This regression was introduced with #1134.

**Comment by gmaurel** [comment]
Fixed 927e4a885d1c41c1e78f8d0aabb5111fd0cdb62e, Dec  5 14:54:30 2017
